buildsystem/model/golang_control.go

```diff
@@ -0,0 +1,120 @@
+package model
```

Comment:
add documentation to exported types and funcs

Reply:
Done.